src/components/plots/PlotResetButton.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+import { initialPlotConfigStates } from 'redux/reducers/componentConfig/initialState';
+import { savePlotConfig } from 'redux/actions/componentConfig';
+import { LOAD_CONFIG } from 'redux/actionTypes/componentConfig';
+
+import _ from 'lodash';
+
+const ResetButton = (props) => {
+  const {
+    experimentId,
+    plotUuid,
+    plotType,
+  } = props;
+
+  const dispatch = useDispatch();
+
+  const [enableReset, setEnableReset] = useState(true);
+  const { config } = useSelector((state) => state.componentConfig[plotUuid] || {});
+
+  const checkConfigEquality = (currentConfig, initialConfig) => {
+    const ignoredFields = {
+      // config fields that are set dynamically on component render
+      // should not be compared to their initial values
+      frequency: ['proportionGrouping', 'xAxisGrouping'],
+      embeddingCategorical: ['axes'],
+      embeddingContinuous: ['shownGene', 'axes'],
+      violin: ['shownGene'],
+      markerHeatmap: ['selectedGenes', 'groupedTracks'],
+      DotPlot: ['selectedGenes'],
```

Comment:
If we change the plotUuid's of a plot this will not work, can you check if you can add the plot names dynamically? Do we have a file with the plot names?

Reply:
This component is a carry over from a previous commit. This is not used anymore in favor of a direct button as implemented in `PlotContainer.jsx`. I'm deleting this,